docker/Dockerfile

```diff
@@ -28,70 +23,30 @@ RUN echo "## Start building" \
             ca-certificates \
             build-essential \
             file \
-            python python-pip \
             python3 python3-pip \
-    \
 &&  echo "## Done"
 
 RUN  echo "## Get EMSDK" \
     &&  git clone https://github.com/emscripten-core/emsdk.git ${EMSDK} \
-    &&  cd ${EMSDK} && git reset --hard ${EMSDK_CHANGESET} \
-    \
-    &&  ./emsdk.py update-tags \
 &&  echo "## Done"
 
 RUN  echo "## Install Emscripten" \
     &&  cd ${EMSDK} \
-    && ./emsdk install ${EMSCRIPTEN_VERSION} \
-    \
+    &&  ./emsdk install ${EMSCRIPTEN_VERSION} \
 &&  echo "## Done"
 
 # This generates configuration that contains all valid paths according to installed SDK
 RUN cd ${EMSDK} \
     &&  echo "## Generate standard configuration" \
-    \
     &&  ./emsdk activate ${EMSCRIPTEN_VERSION} --embedded \
-    &&  ./emsdk construct_env > /dev/null \
+    &&  ./emsdk construct_env \
     &&  cat ${EMSDK}/emsdk_set_env.sh \
-    \
-    # remove wrongly created entry with EM_CACHE, variable will be picked up from ENV
-    &&  sed -i -e "/EM_CACHE/d" ${EMSDK}/emsdk_set_env.sh \
-    # add a link to tools like asm2wasm in a system path
-    # asm2wasm (and friends might be places either in ./upstream of ./fastcomp folder, hence detection is needed)
-    &&  printf "export PATH=$(dirname $(find . -name asm2wasm -exec readlink -f {} +)):\$PATH\n" >> ${EMSDK}/emsdk_set_env.sh \
-    \
-&&  echo "## Done"
-
-# Create a structure and make mutable folders accessible for r/w
-RUN cd ${EMSDK} \
-    &&  echo "## Create .data structure" \
-    &&  for mutable_dir in ${EM_DATA} ${EM_PORTS} ${EM_CACHE} ${EMSDK}/zips ${EMSDK}/tmp; do \
-            mkdir -p ${mutable_dir}; \
-            chmod -R 777 ${mutable_dir}; \
-        done \
-        \
+    &&  chmod -R 777 ${EMSDK} \
```

Comment:
Previously all mutable folders were put inside `.data` folder to avoid changing recursively entire emsdk folder.

Reply:
Do we care if the user has write access to the emsdk directory?
 
I can update to just be the cache directory if we do.

Reply:
Updated to be just "emsdk/cache"  (which is where the ports and the cached libraries live).